themes/digital.gov/layouts/news/card-news.html

```diff
@@ -0,0 +1,109 @@
+{{/* Display a news card for internal or external blog links.
+  External links use both source and source_url to set the logo and external link.
+*/}}
+
+{{/* Unique ID to link article to its title in screen readers. Turns "18f at ten" into "title-18f-at-ten" */}}
+{{ $titleID := (printf "%s%s" "title-" .Title) | urlize }}
+{{ $title := .Title }}
+{{ $kicker := .Params.kicker }}
+{{ $deck := .Params.deck }}
+{{ $summary := .Params.summary }}
+{{ $authors := .Params.authors }}
+{{/* source_url is an external link */}}
+{{ $externalLink := .Params.source_url }}
+{{ $internalLink := .Permalink }}
```

Comment:
After looking at your suggestions, I did some futher refactoring.

- moved variables to top of page, renamed with camelCasing for consistency and added some additional comments